workshops/4x4/index.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: 4x4 Keyboard Soldering Workshop
+subtitle: TBA • Living Computer Museum
+---
+<h2>Build your own 4x4 keyboard</h2>
+  <p>Learn the basics of keyboard electronics, microcontrollers, and soldering. Leave the workshop with your own 4x4 USB keyboard. Before you go, we will flash your keyboard with your choice as a num pad, emoji keyboard (Linux and Windows only), or midi controller. After the workshop you can learn how to program and flash your fully programmable keyboard yourself (or attend one of our not-yet-announced workshops on keyboard firmware). We will provide all tools and supplies.</p>
```

Comment:
Not actually a midi controller right now (though that is possible). It just buzzes the speaker and can be used as a (rather poor) all-in-one midi keyboard.

Making it an actual midi device is doable via qmk, but requires something to interpret the midi signals coming out of the keyboard. That means special software on the computer or an actual midi interpreter in hardware or something.